moler/cmd/at/quectel_lock_nr_earfcn.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+AT+QNWCFG="nr5g_earfcn_lock",(0..32),earfcn1:scs1:...:earfcnN:scsN
+
+AT commands specification:
+google for: Quectel RG50xQRM5xxQ (R11 release and later)
+This is internal Quectel AT command
+(always check against the latest vendor release notes)
+"""
+
+__author__ = 'Jakub Kochaniak'
+__copyright__ = 'Copyright (C) 2023, Nokia'
+__email__ = 'jakub.kochaniak@nokia.com'
+
+import re
+
+from moler.cmd.at.genericat import GenericAtCommand
+from moler.exceptions import ParsingDone
+
+
+class SetQuectelLockNrEarfcn(GenericAtCommand):
+    """
+    Command to lock NR EARFCN. Example output:
+
+    AT+QNWCFG="nr5g_earfcn_lock",1,433970:15
+    OK
+    """
+    def __init__(self, earfcn, scs, connection=None, prompt=None, newline_chars=None, runner=None):
+        """
+        Quectel UE lock one EARFCN.
+
+        :param earfcn: EARFCN number
+        :param scs: SCS number
+        :param connection: Moler connection to device, terminal when command is executed.
+        :param prompt: prompt where we start from.
+        :param newline_chars: Characters to split local lines - list.
+        :param runner: Runner to run command.
+        """
+        super(SetQuectelLockNrEarfcn, self).__init__(connection, operation="execute", prompt=prompt,
+                                                     newline_chars=newline_chars, runner=runner)
+        self.earfcn = int(earfcn)
+        self.scs = int(scs)
+
+    def build_command_string(self):
+        command_prefix = 'AT+QNWCFG='
+        command_values = '"nr5g_earfcn_lock",1,{}:{}'.format(self.earfcn, self.scs) if self.earfcn > 0 \
+            else '"nr5g_earfcn_lock",0'
+        return command_prefix + command_values
+
+    _re_status = re.compile(r'^\s*(?P<STATUS>OK|ERROR)\s*$')
+
+    def on_new_line(self, line, is_full_line):
```

Comment:
I'm wondering if we really need to parsing status - in similar commands like here: https://github.com/nokia/moler/blob/master/moler/cmd/at/set_cell_id_gprs.py we don't do this.  In my opinion parsing status of this command and keep it in results is unnecessary. It brings no new information - moler can itself fail/pass command.